src/System.Private.ServiceModel/src/System/ServiceModel/Dispatcher/OperationFormatter.cs

```diff
@@ -698,6 +715,11 @@ protected override void OnWriteBodyContents(XmlDictionaryWriter writer)
                     }
                 }
 
+                protected override async Task OnWriteBodyContentsAsync(XmlDictionaryWriter writer)
+                {
+                    await _operationFormatter.SerializeBodyContentsAsync(writer, _version, _parameters, _returnValue, _isRequest);
```

Comment:
When the method body solely consists of awaiting another method, just return the Task from that method and remove the async keyword. E.g.:  
```csharp
protected override Task OnWriteBodyContentsAsync(XmlDictionaryWriter writer)
{
    return _operationFormatter.SerializeBodyContentsAsync(writer, _version, _parameters, _returnValue, _isRequest);
}
```